pkg/experiment/sensitivity/factory.go

```diff
@@ -204,10 +204,9 @@ func (f AggressorFactory) getDecorators(name string) isolation.Decorators {
 		decorators := isolation.Decorators{f.l1AggressorIsolation}
 		return decorators
 	case caffe.ID:
-		if RunCaffeWithLLCIsolationFlag.Value() {
-			return isolation.Decorators{f.otherAggressorIsolation}
-		}
 		return isolation.Decorators{}
+	case CaffeAggressorWithIsolation:
```

Comment:
I believe that expclit is more importance that saving a lines - we're safe in case of changing default behavior (and because `CaffeAggressorWithIsolation` is explicitly about running with isolations!)